entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+
+if [ "$SPIDER_TO_RUN" == 'who_iris' ]
+then
+    scrapy crawl who_iris
+elif [ "$SPIDER_TO_RUN" == 'nice' ]
+then
+    scrapy crawl nice
+else
+    echo "Either you did not specified a spider, or the spider you want to run does not exists."
```

Comment:
"Either you did not specify a spider, or the spider you want to run does not exist."